src/AsteroidMining/GameObjectTest.java

```diff
@@ -0,0 +1,104 @@
+package src.AsteroidMining;
+
+import org.junit.jupiter.api.Test;
+
+import java.awt.*;
+import java.util.Random;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class GameObjectTest {
+
+    @Test
+    void getBounds() {
```

Comment:
This file is basically testing getters and setters which is an bad practice. In my opinion, files Game and Handler should be tested instead.